### PR TITLE
feat: add personal, community and event

### PR DESCRIPTION
**Pull request by davideimola**
Hi folks,
I would like to add my personal profile, my community profile and the open source event we are organizing

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2597"></a>

### CODE REVIEW

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, please rename the org JSON file name the same as the one on the GitHub Org name.

```diff
- data/SchrodingerHat.json
+ data/Schrodinger-Hat.json
```

File: data/davideimola.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Davide Imola",
```
[the remark this answers is not in the document]
Comment (eddiejaoude) [reply data/davideimola.json:2]: Ideally yes, but I think it is ok to have them both here in this case, but the downside is one with mismatch GitHub username will block the other

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Comment by Pradumnasaraf** [comment]
Event looks @davideimola amzing. I will submit a CFP for that :)

**Pull request merged**